mslice/presenters/quick_options_presenter.py

```diff
@@ -48,7 +48,7 @@ def _set_axis_options(view, target, model, has_logarithmic, grid):
     setattr(model, target, range)
 
     if has_logarithmic is not None:
-        setattr(model, target[:-5] + 'log', view.log_scale.isChecked())
+        model.change_axis_scale(range, view.log_scale.isChecked())
```

Comment:
On my conda build, entering `10e3` for `vmin` or `vmax` on a `cut plot` `x-axis` gives the following stack trace:
`Traceback (most recent call last):
  File "C:\Repos\GitHub\conda\mslice\mslice\presenters\quick_options_presenter.py", line 30, in <lambda>
    view.ok_clicked.connect(lambda: _set_axis_options(view, target, model, has_logarithmic, grid))
  File "C:\Repos\GitHub\conda\mslice\mslice\presenters\quick_options_presenter.py", line 51, in _set_axis_options
    model.change_axis_scale(range, view.log_scale.isChecked())
TypeError: change_axis_scale() takes 2 positional arguments but 3 were given`